sickbeard/tv.py

```diff
@@ -630,24 +630,24 @@ def getImages(self, fanart=None, poster=None):
     def makeEpFromFile(self, file):
 
         if not ek(os.path.isfile, file):
-            logger.log(str(self.indexerid) + u": That isn't even a real file dude... " + file)
+            logger.log(u"%s: That isn't even a real file %s" % (self.indexerid, file))
             return None
```

Comment:
@miigotu I only changed log format. where I "override built in function names" ?

```
and I told you before about leaving code that has a variable named "file"
logger.log(u"%s: That isn't even a real file %s" % (self.indexerid, file))
if not ek(os.path.isfile, file):
overriding built in function names with badly named variables
```